web/gui/package.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
I'm wondering if there's anyway to consolidate this package.json with our top level one, having two is a bit strange. The other one is for devs who are including chainlink into their truffle project. For that one to stay minimal, we would need to move all of the dependencies here to be devDependencies. Could that still work? Too contorted? Maybe worth experimenting with later on.

Reply:
My original thinking was that because they're isolated apps they would have separate `package.json`'s. I hadn't thought about combining them. I'll play around with the build step to see if it still works with dev dependencies. If it works I don't have a particularly good reason not to combine them.

Thoughts @j16r @dimroc?